src/modules/navigator/navigator_main.cpp

```diff
@@ -898,7 +898,7 @@ Navigator::load_fence_from_file(const char *filename)
  *
  */
 void Navigator::fake_traffic(const char *callsign, float distance, float direction, float traffic_heading,
-			     float altitude_diff, float hor_velocity, float ver_velocity)
+			     float altitude_diff, float hor_velocity, float ver_velocity, int emitter_type)
```

Comment:
```suggestion
			     float altitude_diff, float hor_velocity, float ver_velocity, uint16_t emitter_type)
```

Type should match the size used in the message definition.